starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaCodecBridge.java

```diff
@@ -853,7 +853,7 @@ private int flush() {
     try {
       mFlushed = true;
       mMediaCodec.flush();
-    } catch (IllegalStateException e) {
+    } catch (Exception e) {
```

Comment:
Do we have any general strategy in place to propagate detailed exception info from here and making it available in session data for clients ? 

For example, there are `CreateMediaCodecBridgeResult` and `DequeueInputResult` and `DequeueOutputResult` and `GetOutputFormatResult` that seem to be intended to wrap rich exception results and return it to caller. Perhaps those could be generalized into a common base, and every operation could return details with some `wrapExceptionInfo(e)` call ?